packages/addons/addon-depends/chrome-depends/harfbuzz-icu/package.mk

```diff
@@ -2,7 +2,7 @@
 # Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
-PKG_NAME="harfbuzz"
+PKG_NAME="harfbuzz-icu"
```

Comment:
can we do something like this https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/addons/addon-depends/chrome-depends/chrome-libXcomposite/package.mk ?

so we don't need to maintain two packages